www/securestorage.js

```diff
@@ -24,7 +17,7 @@ var _merge_options = function (defaults, options){
         res[attrname] = defaults[attrname];
     }
     for (attrname in options) {
-        if (res[attrname]) {
+        if (res.hasOwnProperty(attrname)) {
```

Comment:
`if (res[attrname])` could be defined but false positive, you should check for `undefined` or use `hasOwnProperty`